src/Assignment-4/assignment4.html

```diff
@@ -6,12 +6,75 @@
 	<meta http-equiv="Pragma" content="no-cache" />
 	<meta http-equiv="Expires" content="0" />
 	<script src="bundle.js" type="text/javascript"></script>
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
+	<link rel="stylesheet" type="text/css" href="css/assignment4.css">
+	<link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
 </head>
 
 <body>
 	<div id="container">
 
 	<!-- Start your code from here -->
+	<header>
+		<img class="logo" src="images/Logo.jpg" alt = "Dog logo" height="100" width="100" />
+		<nav>
+		  <ul>
+			<li><strong> Home</strong></li>
+			<li><strong>Things</strong></li>
+			<li><strong>Stuff</strong></li>
+		  </ul>
+		</nav>
+	  </header>
+  
+	  <section id="hero" >
+		  <div >
+		<h2>A Site About Some Dogs</h2>
+		<img src="images/Dog1.jpg" alt="hero dog"/>
+		<a href="http://corgis-in-space.tumblr.com/" target="_blank">What are Dogs</a>
+	   </div>
+	  </section>
+  
+	  <section>
+		<article class="three-col">
+			<h3><stong>Home</stong></h3>
+      <img src="images/Dog2.jpg" alt="first image"/>
+      <p>
+        Nap all day Gate keepers of hell yet stares at human while pushing stuff off a table. Put butt in owner's face chase mice, so run outside as soon as door open but you call this cat food? stare at the wall, play with food and get confused by dust.
+      </p>
+      <a href="https://en.wikipedia.org/wiki/Dog">Go Home</a>
+
+		</article>
+		<article class="three-col">
+			<h3><stong>Things</stong></h3>
+			<img src="images/Dog3.jpg" alt="second image"/>
+			<p>
+			  Soft kitty warm kitty little ball of furr. Hunt anything that moves meowing non stop for food stand in front of the computer screen knock dish off table head butt cant eat out of my own dish, then cats take over the world hide when guests come over.
+			</p>
+			<a href="http://www.dog-adoption-and-training-guide.com/about-dogs.html">More Things</a>
+	  
+		</article>
+		<article class="three-col">
+			<h3><stong>Stuff</stong></h3>
+			<img src="images/Dog4.jpg" alt="third image"/>
+			<p>
+			  Destroy the blinds chase imaginary bugs, so lie on your belly and purr when you are asleep. Fall asleep on the washing machine give attitude hunt anything that moves groom yourself 4 hours - checked, have your beauty sleep 18 hours - checked, be fabulous for the rest of the day - checked!
+			</p>
+			<a href="http://www.livescience.com/13305-facts-dog-breeds-genetics-pets.html">Some stuff</a>
+		</article>
+	  </section>
+  
+	  <footer>
+		<h4>Learn About Dogs Online</h4>
+		<ul>
+			<li><img src="images/FB icon.jpg"/></li>
+			<li><img src="images/twitter icon.jpg"/></li>
+			<li><img src="images/Insta icon.jpg"/></li>
+			<li><img src="images/LinkedIn icon.jpg"/></li>
+		</ul>
+		<a href="https://github.com/turingschool-examples/introductory-static-site" target="_blank" class="link">See this on GitHub</a>
```

Comment:
this a tag should not look like button can you please change it?
![Assignment-4 2](https://user-images.githubusercontent.com/57532391/148639669-173f40d7-d9fd-46dc-b958-a055284c2e05.png)